src/components/inputs/z-combobox/index.tsx

```diff
@@ -147,6 +157,55 @@ export class ZCombobox {
     this.selectedCounter = this.itemsList.filter((item) => item.checked).length;
   }
 
+  private resetInputTabIndex(): void {
+    this.itemsList.forEach(function (item: ComboItem) {
+      this.element.shadowRoot.querySelector(`#combo-checkbox-combo_1-${item.id}`).setAttribute("tabindex", "-1");
+    }, this.element);
+  }
+
+  private setInputTabIndex(itemId, index): number {
+    if (index === 0) {
+      return 0;
+    }
+
+    return this.getFocusedItemIndex(itemId) === index ? 0 : -1;
+  }
+
+  private getFocusedItemIndex(itemId): number {
+    return itemId.substring(itemId.indexOf("_") + 1);
+  }
+
+  private arrowsSelectNav(e: KeyboardEvent, key: number): void {
+    const arrows = [KeyboardCode.ARROW_DOWN, KeyboardCode.ARROW_UP];
+    if (!arrows.includes(e.key as KeyboardCode) || !this.isopen) {
```

Comment:
Se leggo bene la documentazione, andrebbe gestito il caso in cui la tendina è chiusa e premo la freccia in basso, che in quel caso dovrebbe aprire la tendina.

Reply:
Il bottone, se interpreto bene il senso delle combobox non  fa parte del comportamento del componente.
è un  po' come se il bottone  aprisse/chiudesse la combobox, con lisbox sempre visibile. 
In ogni  caso sull'header ha senso avere il button _togglabile_  con le frecce, alla fine è un dropdown. 
Ho integrato  anche con `aria-expanded` per completezza